src/mds/cephfs_features.cc

```diff
@@ -0,0 +1,73 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include <array>
+#include <sstream>
+#include "cephfs_features.h"
+#include "mdstypes.h"
+
+static const std::array feature_names
+{
+  "reserved",
+  "reserved",
+  "reserved",
+  "reserved",
+  "reserved",
+  "jewel",
+  "kraken",
+  "luminous",
+  "mimic",
+  "reply_encoding",
+  "reclaim_client",
+  "lazy_caps_wanted",
+  "multi_reconnect",
+  "deleg_ino",
+  "metric_collect",
+};
+
+static const std::array feature_aliases
+{
+  std::make_pair("nautilus", "multi_reconnect"),
+  std::make_pair("octopus", "deleg_ino"),
+  std::make_pair("pacific", "metric_collect"),
```

Comment:
What do we do when there's multiple features in a release? Just pick one? I'm not sure these aliases are useful anymore since we don't want users to select a release anymore.

Reply:
select the last one.

I still think selecting release is more user friendly. features bits are more about mds->client protocol, not what user/admin cares about. It also requires some knowledge of mds internal to understand what each feature is.

Reply:
Last one is too arbitrary. Select all of them instead.